base-template.sh

```diff
@@ -406,10 +407,7 @@ is_trusted_repo() {
 #####################################################
 execute_opt_in_checks() {
     # get hash of the hook contents
-    if ! MD5_HASH=$(md5 -r "$HOOK_PATH" 2>/dev/null); then
-        MD5_HASH=$(md5sum "$HOOK_PATH" 2>/dev/null)
-    fi
-    MD5_HASH=$(echo "$MD5_HASH" | awk "{ print \$1 }")
+    SHA_HASH=$(git hash-object "$HOOK_PATH" 2>/dev/null)
```

Comment:
Oh, this means people will have to opt-in to all their existing hooks in all their repositories?
Maybe a pre-install warning would be nice if that's easy to do.
Or perhaps we could revert this and deal with it in a follow-up PR? (maybe we could also transform these automatically)

Reply:
See my lates commit